aiohttp/http_parser.py

```diff
@@ -547,18 +548,17 @@ def feed_data(self, chunk, SEP=b'\r\n', CHUNK_EXT=b';'):
                     required = self._chunk_size
                     chunk_len = len(chunk)
 
-                    if required >= chunk_len:
+                    if required > chunk_len:
```

Comment:
If `required == chunk_len`, then `self._chunk_size == 0` and the "else" part does what is needed. 
So lines 552 and 553 looked like duplicated code, and deduplicating this allowed me to put `self.payload.end_http_chunk_receiving()` at only one place in the code.

But the code will continue after the 'else' as opposed to the previous version where (False, None) was returned immediately. So the behaviour is not exactly the same even if the same result will be eventually returned.

So imo it looks better but if you want me to revert to the previous version tell me.

Also I'm going to remove the useless blank line(554)